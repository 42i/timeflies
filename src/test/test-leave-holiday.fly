day 2012-07-26, phol; Badger's Day
leave-days 2012-07-24 2012-07-31; watch grass grow
leave-days 2012-08-01 2012-08-08; watch Aurora Borealis
day 2012-08-02, phol; Troll's Day