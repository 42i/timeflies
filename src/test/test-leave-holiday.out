Time at work overview (all):
     when        worked   leave    sick    must
2012-07-24 Tue: ----.--    8.00 ----.--    8.00 watch grass grow
2012-07-25 Wed: ----.--    8.00 ----.--    8.00 watch grass grow
2012-07-26 Thu: ----.-- ----.-- ----.-- ----.-- Badger's Day
2012-07-27 Fri: ----.--    8.00 ----.--    8.00 watch grass grow
  week 2012-30: ----.--   24.00 ----.--   24.00
2012-07-30 Mon: ----.--    8.00 ----.--    8.00 watch grass grow
2012-07-31 Tue: ----.--    8.00 ----.--    8.00 watch grass grow
 month 2012-07: ----.--   40.00 ----.--   40.00
2012-08-01 Wed: ----.--    8.00 ----.--    8.00 watch Aurora Borealis
2012-08-02 Thu: ----.-- ----.-- ----.-- ----.-- Troll's Day
2012-08-03 Fri: ----.--    8.00 ----.--    8.00 watch Aurora Borealis
  week 2012-31: ----.--   32.00 ----.--   32.00
2012-08-06 Mon: ----.--    8.00 ----.--    8.00 watch Aurora Borealis
2012-08-07 Tue: ----.--    8.00 ----.--    8.00 watch Aurora Borealis
2012-08-08 Wed: ----.--    8.00 ----.--    8.00 watch Aurora Borealis
  week 2012-32: ----.--   24.00 ----.--   24.00
 month 2012-08: ----.--   40.00 ----.--   40.00
         total: ----.--   80.00 ----.--   80.00
     when        worked   leave    sick    must
